src/components/Atoms/Icon/Icon.tsx

```diff
@@ -12,6 +13,7 @@ export type TIconContainerProps = {
 
 export type TIcon = {
     name: keyof typeof icons;
+	id?: string;
```

Comment:
@Ivana-Jovic Zašto je bio potreban id ovde?

Reply:
@AleksaStevic Zbog onog text buttona, koji ima ikonicu koja se pomera na hover, focus itd